src/main/java/org/waarp/common/crypto/ssl/WaarpSecureKeyStore.java

```diff
@@ -308,6 +390,7 @@ public void initTrustStore(String _trustStoreFilename, String _trustStorePasswd,
         try {
             inputStream = new FileInputStream(trustStoreFilename);
             keyTrustStore.load(inputStream, getKeyTrustStorePassword());
+            validateKeyStore(keyTrustStore);
```

Comment:
Return value should not be discarded: invalid certificates cannot be trusted and transfers from clients which use these certificates must be forbidden.

A good way would be to filter out all invalid certificates from the truststore and only keep valid ones...